cli/src/display.rs

```diff
@@ -200,3 +203,50 @@ pub fn println_transaction(
         }
     }
 }
+
+pub fn output_account(
+    output_format: &OutputFormat,
+    pubkey: &Pubkey,
+    account: &Account,
+    use_lamports_unit: bool,
+    account_data_output: AccountDataOutputFormat,
+) -> String {
+    let rpc_account = match account_data_output {
+        AccountDataOutputFormat::Base58 => RpcAccount::encode_with_base58(account.clone()),
+        AccountDataOutputFormat::Hex => RpcAccount::encode_with_base58(account.clone()),
```

Comment:
encode_with_base58 for AccountDataOutputFormat::Hex?

Reply:
Nice catch! the last pushed commit is still being baked... ;)